models/user.js

```diff
@@ -7,8 +7,30 @@ const UserSchema = new Schema({
     type: String,
     required: true,
   },
+  displayName: {
+    type: String,
+  },
+  profileImg: {},
```

Comment:
What about the type here?

Reply:
i was not too sure about how the type is supposed to look exactly just yet, so i decided to leave it for when i do the image upload task